CustomApps/lyrics-plus/index.js

```diff
@@ -162,8 +162,13 @@ class LyricsContainer extends react.Component {
     async fetchColors(uri) {
         let prominent = 0;
         try {
-            const colors = await CosmosAsync.get(`hm://colorextractor/v1/extract-presets?uri=${uri}&format=json`);
-            prominent = colors.entries[0].color_swatches[4].color;
+            // const colors = await CosmosAsync.get(`hm://colorextractor/v1/extract-presets?uri=${uri}&format=json`);
+            // prominent = colors.entries[0].color_swatches[4].color;
+            // hermes deprecated 1.1.81 onwards
```

Comment:
Why did you choose to leave this commented and not delete it?

Reply:
That was just me being lazy, so I wouldn't forget how it worked before, should I remove it?

Reply:
Yeah, as I said on the review: "If we need the previous code, we'll check the git history, no need to leave old code commented 😉"